eng/Versions.props

```diff
@@ -2,8 +2,8 @@
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     
-    <VersionPrefix>2.9.2</VersionPrefix>
-    <PreReleaseVersionLabel>beta2</PreReleaseVersionLabel>
+    <VersionPrefix>2.9.3</VersionPrefix>
```

Comment:
This seems a wrong merge resolution - version should be 3.0.0

Reply:
The merge conflict has not been resolved yet. I can update the PR.